Duplicati/GUI/Duplicati.GUI.TrayIcon/Program.cs

```diff
@@ -319,14 +314,6 @@ private static TrayIconBase RunTrayIcon(string toolkit)
 
         [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
         private static TrayIconBase GetWinformsInstance() { return new Win32Runner(); }
-#if __MonoCS__ || __WindowsGTK__ || ENABLE_GTK

```

Comment:
I think we only want to remove the `__MonoCS__` part of the `#if` condition.  The rest of this code should remain.

Reply:
Alright. Will restore that then.